docs/release-notes/NuGet-6.8.md

```diff
@@ -0,0 +1,164 @@
+
+* Add [protocolVersion argument](../reference/cli-reference/cli-ref-sources.md#options) to nuget source add - [#9170](https://github.com/NuGet/Home/issues/9170)
+
+* Conditional package updating is respected in Visual Studio [#5420](https://github.com/NuGet/NuGet.Client/pull/5420)
+
+### Breaking changes
+
+* Remove the NuGetOperationType from NuGet.PackageManagement, use NuGetProjectActionType instead - [#12866](https://github.com/NuGet/Home/issues/12866)
+
+* Changing PackageVulnerabilityInfo severity from int to enum - [#12781](https://github.com/NuGet/Home/issues/12781)
+
+* Add nullable annotations to NuGet.Common - [#12775](https://github.com/NuGet/Home/issues/12775)
+
+* Obsolete Clone methods on immutable types - [#12669](https://github.com/NuGet/Home/issues/12669)
+
+### Issues fixed in this release
+
+* NuGetAudit should not warn when no vulnerability data is available - [#12875](https://github.com/NuGet/Home/issues/12875)
+
+* NuGetAudit: read vulnerability files with System.Text.Json - [#12855](https://github.com/NuGet/Home/issues/12855)
+
+* `PackageSourceMapping` API doesn't follow best practices for returning lists - [#12794](https://github.com/NuGet/Home/issues/12794)
+
+* Signing:  enable `X509Chain.Build(...)` retry behavior by default - [#12592](https://github.com/NuGet/Home/issues/12592)
+
+* NuGetAudit should check direct PackageReferences by default - [#12590](https://github.com/NuGet/Home/issues/12590)
+
+* NuGetAudit should be on by default with the .NET 8 SDK - [#12568](https://github.com/NuGet/Home/issues/12568)
+
+* Remove "Checking compatibility..." log messages from RestoreTask - [#10383](https://github.com/NuGet/Home/issues/10383)
+
+* 16.10: remove package source 1.0 service. remove obsolete APIs (in nuget.configuration that we added in 16.8) - [#10015](https://github.com/NuGet/Home/issues/10015)
+
+* Add more logging to NuGetSdkResolver - [#11445](https://github.com/NuGet/Home/issues/11445)
+
+* Upgrade Newtonsoft.Json reference to 13.0.3 - [#12858](https://github.com/NuGet/Home/issues/12858)
+
+* Add an API for checking vulnerability during packages.config restore - [#12852](https://github.com/NuGet/Home/issues/12852)
+
+* VS Options add/remove package source icons aren't using VS2022 styling - [#12840](https://github.com/NuGet/Home/issues/12840)
+
+* Package Source Mapping utility always appends package ID - [#12839](https://github.com/NuGet/Home/issues/12839)
+
+* NuGetSdkResolver loads global.json multiple times during project load - [#12819](https://github.com/NuGet/Home/issues/12819)
+
+* dotnet list package doesn't list requested versions when using CPM - [#12765](https://github.com/NuGet/Home/issues/12765)
+
+* NuGet doesn't restore RID-specific assets for Microsoft.DotNet.ILCompiler - [#12757](https://github.com/NuGet/Home/issues/12757)

```

Comment:
This shows a problem using issue titles, rather than PR titles.  The fix wasn't specific to the Microsoft.DotNet.ILCompiler package. The PR title is better

```suggestion
* Fix case sensitivity of runtime dependency sets during merge - [#12757](https://github.com/NuGet/Home/issues/12757)
```

Reply:
Realistically neither are always gonna work great for release notes. We can fix this one in particular.